src/main/webapp/index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="UTF-8">
     <title>Play User Data Beetle</title>
+    <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getDatabases()">
     <h1>Play User Data Beetle</h1>
     <p>Click <a href="/example-db">here</a> to view data queried from an example Cloud Spanner database.</p>
+    <p>please select the database you would like to use</p>
+
+    <div id="select-db">
+        <select id="list-databases" value="list-databases" class="select">
```

Comment:
What does setting the value to list-databases do? It isn't even an option in the select.

Reply:
That is true, I guess I'm used to examples having values for many elements even if they aren't necessary, but I removed the value.